taxcalc/taxcalcio.py

```diff
@@ -47,11 +47,11 @@ class TaxCalcIO(object):
         or string is name of optional ASSUMP file.
 
     growdiff_response: Growdiff object or None
-        growdiff_response Growdiff object used in dynamic analysis;
-        must be None when conducting static analysis.
+        growdiff_response Growdiff object used only in dynamic analysis;
+        must be None when not conducting dynamic analysis.
```

Comment:
> I thought your taxonomy was: static, partial-equilibrium(ie, behavior), and dynamic analysis, right?

That's not right. My point was that partial-equilibrium analysis should be considered dynamic analysis since it changes GNP, and in the tax-scoring world anything that changes GNP is called dynamic. Will give you a call shortly. 

Note that this is the taxonomy we use on TaxBrain since partial equilibrium analysis is available from the dynamic landing page.